pub/culture.html

```diff
@@ -0,0 +1,23 @@
+<header>Our Culture</header>
+
+<p>If you walk into The Iron Yard on any given afternoon, 
+you’re as likely to see an engineering team architecting software as you are to see a group enjoying craft 
+brews over ping pong and video games.</p>
+
+<p>We’ve been told we’re a “work hard, play hard” crew, and we couldn’t agree more. Our standards for producing awesome stuff are extremely high,
+but so is our capacity to truly enjoy life alongside the people we work with.</p>
```

Comment:
lol just for the record I did days of keybr.io and my max wpm is 106 and average is 80 wpm